### PR TITLE
README add logo & badges

### PR DESCRIPTION
**Pull request by KoenBal**
### Description
As part of the project README redesign
added logo
added shields aka badges

Rationale: For some users the celo monorepo could be the first impression they have of celo. A 
A nice README is a good way to help people engage in the project as well. A project with nice README and screenshots will get the attention of users better since it’s a direct way to explain why this project matters, and why people should use and contribute to the project. Good README should also include enough details to help a new user get started, e.g. how to compile, how to install, and how to start integrating.

README educational content I used.
[A beginners guide to writing kickass readmes](https://medium.com/@meakaakka/a-beginners-guide-to-writing-a-kickass-readme-7ac01da88ab3)
[AWESOME READMEs](https://github.com/matiassingers/awesome-readme)

### Tested
tested links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,53 @@
...
+![GitHub repo size](https://img.shields.io/github/repo-size/celo-org/celo-monorepo)
+[![GitHub](https://img.shields.io/github/license/celo-org/celo-monorepo?color=blue)](https://github.com/celo-org/celo-monorepo/blob/master/LICENSE)
+
+<!-- row 2 - links & profiles -->
+[![Website celo.org](https://img.shields.io/website-up-down-green-red/https/celo.org.svg)](https://celo.org)
+[![Blog](https://img.shields.io/badge/blog-up-green)](https://medium.com/celohq)
+[![docs](https://img.shields.io/badge/docs-up-green)](https://docs.celo.org/)
+[![Youtube](https://img.shields.io/badge/YouTube%20channel-up-green)](https://www.youtube.com/channel/UCCZgos_YAJSXm5QX5D5Wkcw/videos?view=0&sort=p&flow=grid)
+[![forum](https://img.shields.io/badge/forum-up-green)](https://forum.celo.org)
+[![Discord](https://img.shields.io/badge/Discord-29%20online-brightgreen)](https://discord.gg/JvJ66Wc)
```
Comment (timmoreton) [inline README.md]: I just enabled widget. Try `https://img.shields.io/discord/600834479145353243`

File: README.md [hunk 2]
```diff
@@ -1,13 +1,53 @@
...
+[![Twitter Follow](https://img.shields.io/twitter/follow/celohq?style=social)](https://twitter.com/celohq)
+[![Subreddit subscribers](https://img.shields.io/reddit/subreddit-subscribers/CeloHQ?style=social)](https://www.reddit.com/r/CeloHQ/)
+
+<!-- row 3 - detailed status -->
+[![GitHub pull requests by-label](https://img.shields.io/github/issues-pr-raw/celo-org/celo-monorepo)](https://github.com/celo-org/celo-monorepo/pulls)
+[![GitHub Issues](https://img.shields.io/github/issues-raw/celo-org/celo-monorepo.svg)](https://github.com/celo-org/celo-monorepo/issues)
+[![GitHub issues by-label](https://img.shields.io/github/issues/celo-org/celo-monorepo/1%20hour%20tasks)](https://github.com/celo-org/celo-monorepo/issues?q=is%3Aopen+is%3Aissue+label%3A%221+hour+tasks%22)
+[![GitHub issues by-label](https://img.shields.io/github/issues/celo-org/celo-monorepo/betanet-blocker)](https://github.com/celo-org/celo-monorepo/issues?q=is%3Aopen+is%3Aissue+label%3Abetanet-blocker)
```
Comment (timmoreton) [inline README.md]: pls can you replace this with `betanet-phase-2` and add one for `betanet-phase-3` ?

**Review by timmoreton — COMMENTED** [review]
Thanks for adding these! see comments

**Comment by timmoreton** [comment]
BTW we use the assignee field for managing PRs. We should explain that better :-) Set me as assignee again when you want me to take a look.

**Comment by KoenBal** [comment]
> BTW we use the assignee field for managing PRs. We should explain that better :-) Set me as assignee again when you want me to take a look.

I've been looking to do that its mentioned in the contributing guidelines but I've my confirmation now from help (see below). I don't think I or any other "non-collaborator" is able to assign someone.

Help says: 
> Assigning issues and pull requests to other GitHub users
> Assignees clarify who is working on specific issues and pull requests.
> Anyone with write permissions to a repository can assign issues and pull requests.

My issues look like this:
image

Solution: #2111

File: README.md [hunk 3]
```diff
@@ -33,7 +70,7 @@ See [Developer's Guide](https://docs.celo.org/) for full details about the desig
 
 ## Issues
 
-See the [issue backlog](https://github.com/celo-org/celo-monorepo/issues) for a list of active or proposed tasks. Feel free to create new issues to report bugs and/or request features. Please add labels to your issues, tagging the appropriate package/area.
+See the [issue backlog](https://github.com/celo-org/celo-monorepo/issues) for a list of active or proposed tasks. Feel free to create new issues to report bugs and/or request features.
```
Comment (koenprivate) [inline README.md:94]: "non-collaborators" are not able to add labels I think.
Maybe the Issue template could have categories?

Comment (koenprivate) [reply README.md:94]: See #2111

**Pull request closed without merging**